### ISSUE TITLE
TypeError zodToJsonSchema is not a function

### ISSUE THREAD

**Issue by Adammatthiesen** [HiDeoo/astro-content-devtools#2]
### Describe the bug

TypeError: zodToJsonSchema is not a function
    at Module.parseAstroCollections (E:\git-repo\mxyz-support\node_modules\astro-content-devtools\src\libs\content.ts:15:23)
    at async eval (E:\git-repo\mxyz-support\node_modules\astro-content-devtools\src\AstroContentDevtools.astro:12:42)
    at async bufferHeadContent (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/runtime/server/render/astro/render.js:106:25)
    at async renderToReadableStream (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/runtime/server/render/astro/render.js:46:5)
    at async renderPage (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/runtime/server/render/page.js:29:12)
    at async renderPage (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/core/render/core.js:50:20)
    at async #tryRenderRoute (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/core/pipeline.js:110:18)
    at async DevPipeline.renderRoute (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/core/pipeline.js:58:20)
    at async handleRoute (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/vite-plugin-astro-server/route.js:248:18)
    at async run (file:///E:/git-repo/mxyz-support/node_modules/astro/dist/vite-plugin-astro-server/request.js:50:14)

### To Reproduce

followed install instruction
displays console error:

05:41:53 [ERROR] zodToJsonSchema is not a function
  Stack trace:
    at E:\git-repo\mxyz-support\node_modules\astro-content-devtools\src\libs\content.ts:15:23    [...] See full stack trace in the browser, or rerun with --verbose.    [...] See full stack trace in the browser, or rerun with --verbose. 

### Expected behavior

when adding <AstroContentDevtools collections={collections} /> to body i expect to see atleast a working website but am presented with this error

### How often does this bug happen?

Every time

### System Info

- astro: "^4.0.8",
- astro-content-devtools: "^0.0.4"
- @astrojs/solid-js: "^3.0.3"
- solid-js: "^1.8.8"

**Comment by Adammatthiesen** [comment]
Just in case it helps... this is the repo im trying to deploy the tool into... https://github.com/Adammatthiesen/mxyz-support

**Comment by HiDeoo** [comment]
Thanks for your report :raised_hands:

I just released a [new version](https://github.com/HiDeoo/astro-content-devtools/releases/tag/v0.1.0) of the package that should fix this issue and bring compatibility with Astro v4.

I took the liberty to test it on your project and the issue seems to be fixed:

<img width="1698" alt="SCR-20240104-pmlc" src="https://github.com/HiDeoo/astro-content-devtools/assets/494699/acb68398-1ff9-4121-9814-46bb0141d41d">

A few notes:

- You will also need to install the latest version of `@astrojs/solid-js`.
- Astro `data` content collections are not supported yet.
- Now that Astro 4.0 has a built-in [Dev Toolbar](https://astro.build/blog/astro-4/#the-astro-dev-toolbar), this package should be refactored to a Dev Toolbar App. Not sure if someone is already working on a similar Dev Toolbar App or if I will get the bandwidth to refactor this package in the near future.
